x-pack/test_serverless/functional/test_suites/security/ftr/cloud_security_posture/compliance_dashboard.ts

```diff
@@ -39,7 +39,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     },
   ];
 
-  describe('Cloud Posture Dashboard Page', function () {
+  describe.skip('Cloud Posture Dashboard Page', function () {
```

Comment:
as discussed in the thread, let's try adding `{ product_line: 'security', product_tier: 'essentials' }` to the config, this should make the test pass